README.md

```diff
@@ -73,21 +73,21 @@ The Valhalla organization is comprised of several library modules each responsib
 
 ## Documentation
 
-Documentation is stored in the `docs/` folder in this GitHub repository. It can be viewed at [valhalla.readthedocs.io](https://valhalla.readthedocs.io/).
+Documentation is stored in the `docs/` folder in this GitHub repository. It can be viewed at [valhalla.github.io/valhalla](https://valhalla.github.io/valhalla).
 
 ## Installation
 
 To run Valhalla locally or your own server, we recommend using our Docker image. Checkout the `run-*` docker containers here: https://hub.docker.com/r/valhalla/valhalla/tags. Also, there's a [community Docker image](https://github.com/gis-ops/docker-valhalla) with more "magic" than the native one.
 
-If you want to build Valhalla from source, follow the [documentation](docs/building.md).
+If you want to build Valhalla from source, follow the [documentation](https://valhalla.github.io/valhalla/building/).
 
-For more information on binaries, see [Command Line Tools](#command-line-tools) section below and the [docs](docs).
+For more information on binaries, see [Command Line Tools](#command-line-tools) section below and the [docs](https://valhalla.github.io/valhalla).
 
 ## Contributing
 
 We :heart: contributions to Valhalla. They could be non-technical, e.g. translations into other languages via [Transifex](https://www.transifex.com/valhalla/valhalla-phrases/locales-en-us-json--transifex/) or documentation improvements, or technical ones like bug fixes or feature implementations. It's important to open an issue before setting out to work on a PR.
 
-Ideally, get familiar with our [Contribution guidelines](./CONTRIBUTING.md) first.
+Ideally, get familiar with our [Contribution guidelines](https://github.com/valhalla/valhalla/blob/master/CONTRIBUTING.md) first.
```

Comment:
some are docs links like `building` above but this one and the run routes readme link to markdown on the master branch. this is a bit confusing to me is it because docs generation doesn't docs-ify those? or should these markdown links be github.io docs-ified links?

Reply:
Right, it can't link them relatively because it doesn't have access to anything outside the `./docs/docs` folder, see its branch: https://github.com/valhalla/valhalla/tree/gh-pages/. Symlinking wouldn't work here either, then the github.com version wouldn't find the right doc.

Reply:
Or well, with symlinking we could put the absolute URL of the GH Pages version. I don't really mind either way.

Reply:
oh i just wondered if the make docs would turn the markdown into html to be rendered nicely without the built in github viewer but it seems like not? or maybe not by default? i am definitely not up to date on how the docs are generated. like why doesnt `https://valhalla.github.io/valhalla/contributing` get generated from the markdown? oh i see we have to change it in the yaml below and if we did that then we could have a nice html version of the markdown?

Reply:
> if the make docs would turn the markdown into html to be rendered nicely without the built in github viewer

Oh yes, sure it does. If not, we couldn't verify locally. The "github viewer" doesn't do anything at all actually, other than pointing the URL to the right path on their server, the rest is all mkdocs: https://github.com/valhalla/valhalla/blob/gh-pages/index.html. 

Again, it cannot view anything that's outside of `./docs/docs` and CONTRIBUTING.md is in master's root. If we'd like it in the docs HTML version, we'd have to symlink as we did with the root README. We can do that and then put the absolute URL of the docs version, so when someone clicks it on github.com it'll forward to github.io.